api/schemas/report.json

```diff
@@ -136,9 +136,8 @@
         },
 
         "kind": {
```

Comment:
We're updating the report schema version number for Puppet 5 anyway - does it make sense to just entirely rip out this enumeration?